recipes/mpc-obscodes/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "mpc-obscodes" %}
+{% set version = "2025.4.22" %}
+{% set python_min = "3.9" %}
```

Comment:
We only need to set `python_min` if the min version is higher than conda-forge's min. 
```suggestion

```